docs/modules/ROOT/pages/component-loki.adoc

```diff
@@ -0,0 +1,21 @@
+= Loki
+
+[abstract]
+Loki can be used to process and store Couchbase logs.
+
+== Overview
+
+https://grafana.com/oss/loki/[Loki^] is the https://prometheus.io/[Prometheus^]-inspired open source, multi-tenant log aggregation system. Loki is used to process and store collected Couchbase logs.
+
+Couchbase cluster generates logs of various operations which is important for monitoring and investigation purposes. These logs need to be parsed in order to search and aggregate them based on keys. Fluent bit forwards the couchbase logs to loki which enables processing and storage of logs.
+
+image:component-loki.png[]
+
+== Get Started
+
+Loki is managed by the xref:architecture.adoc[Couchbase Monitoring Observability Stack]. Loki is enabled by default in the CMOS stack, however, fluent bit needs to be configured for the logs to reach loki. Depending upon the type of deployment you need to enable fluent bit.
```

Comment:
Capitalise Fluent Bit.

This likely wants to link to the other PR adding the FB deployment details once that is in.

Reply:
>This likely wants to link to the other PR adding the FB deployment details once that is in.
Yes, for now I am resolving the comment for now as that Pr is not in yet